pandas/tests/groupby/test_apply.py

```diff
@@ -638,24 +656,37 @@ def test_func(x):
     tm.assert_frame_equal(result, expected)
 
 
-def test_groupby_apply_none_first():
+@pytest.mark.parametrize("as_index", [True, False])
```

Comment:
Can you make `as_index` a fixture in the groupby conftest? I think will have generic utility in this set of tests